Store data to local storage

Elhambasir opened the pull request:
Save data entered by the user to local storage and retrieve it when the user reloads the web page.

Elhambasir committed:
Create the process of storing data to local storage and retrieve it back

Elhambasir committed:
Fix linter error

Elhambasir committed:
Fix linter error

Elhambasir committed:
Fix linter error

Elhambasir committed:
Fix linter error

Nelofarzabi committed:
Update index.html

didierganthier commented on script.js:
```diff
@@ -30,3 +30,38 @@ btnMenuItem2.onclick = function show() {
+  event.preventDefault();
+});
+
+const btnSubmit = document.querySelector('.btnSubmit');
+btnSubmit.onclick = function storeData() {
+  const name = document.querySelector('#name').value;
+  const comment = document.querySelector('#comment').value;
+  const email = document.querySelector('#email').value;
+  const userInfo = {
+    name,
+    email,
+    comment,
+  };
+  const string = JSON.stringify(userInfo);
+  localStorage.setItem('userInfo', string);
+};
```
Well done on using `localStorage` 💯 but for now the data is only getting stored in `localStorage` once the `form` is submitted 🚀 yet the requirement states that after filling the form, and not submitting it, the entered values should be saved in the browser. 🕸️ So even if the user started typing and the browser closed for some reason, the user should still be able to retrieve the data. I suggest looking into [onchange event](https://www.w3schools.com/jsref/event_onchange.asp) 🔗

didierganthier reviewed (changes requested):
Hi Team,

🎉 Good job so far! You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.

<img src="https://media.giphy.com/media/3oEduMcvxTNPlGuhH2/giphy.gif" />

## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- [ ] Nice job 👍🏽 but make sure to add all the contributors of the project in the `Author` section of the README file. 👥 
- [ ] How about we update the Tech Stack on the README to add `JS` now that we're cool enough to use JS 😎 

- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

Elhambasir committed:
Edit local storage process

Elhambasir committed:
Merge branch 'main' into Local_storage

didierganthier reviewed (changes requested):
Hi Team,

🎉 Great job on addressing the feedback from the previous review! You've made some solid progress in implementing the requested changes. However, there are still a few areas that need some attention before we can move on to the next project.

<img src="https://media.giphy.com/media/e56n4Z0mmkxZS/giphy.gif" />

## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- [ ] Nice job on implementing the changes I requested 💯 but make sure to also take these I pointed out earlier into consideration 👇🏽 
![Screenshot 2023-03-10 at 07 33 39](https://user-images.githubusercontent.com/45739963/224317117-480da6d0-c5a3-41d0-a32b-f89c7ddc1c4b.png)

- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

Elhambasir committed:
Update README.md

Add Nelofarzabi as a contributor in the README.md contributions section, and add JAVASCRIPT to Tech Stack in the README.md file.

Elhambasir committed:
Update README.md

Add all contributors to the README.md file contributions section.  Add javascript to tech stack.

frank1738 reviewed (approved):
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<img src="https://media3.giphy.com/media/tIeCLkB8geYtW/giphy.gif"/>

## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Data is being preserved correctly 👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
